scheduling_strawman.py

```diff
@@ -0,0 +1,99 @@
+"""
+The scheduler is a very useful tool. It requires the input of observing blocks
+and the time_range. Observing blocks consist of a target, a duration and constraints.
+"global" constraints can also be defined and can significantly speed up the scheduler.
+"""
+
+# a few packages are needed to set up the proper objects
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.time import Time
+import astroplan
+from astroplan.constraints import *
+from astroplan import Observer, FixedTarget
+from astroplan.scheduling import *
+
+# create the time frame during which you observing
+schedule_start = Time.now()+4*u.hour
+schedule_end = schedule_start+24*u.hour
+
+# define the observer
+mro = Observer.at_site('mro')
+
+# we also want to define how quickly the telescope can transition
+# between different targets, filters, instruments etc.
+# for using a telescope that slews at 1 degree/second,
+# takes 2 minutes to change filters, and 10 to change instruments
+slew_rate = 1*u.deg/u*second
+transitioner = Transitioner(slew_rate, {filter: 2*u.minute, instrument: 10*u.minute})
```

Comment:
I *think* you mean for ``filter`` and ``instrument`` to be strings, right?  E.g., ``{'filter': 2*u.minute, 'instrument': 10*u.minute})``?

Also, in some telescopes, the time to change something might depend on what you're changing from.  For example, it might take longer to switch from V to J than from V to B.  I don't think you necessarily need to build that in *now*, but it something we should at least consider how it might work.  The simplest thing would just be to let the user pass in a dictionary with the two states and the time.  E.g. something like this:
```
filter_times = {None:2*u.minute, ('V', 'J'): 3*u.minute,  ('V', 'B'): 1.5*u.minute}
transitioner = Transitioner(slew_rate, {'filter': filter_times})  
# In the above example None would presumably be the default.  Could also be a special string like ``"default"``
```